src/unit_tests/cache_tests.cpp

```diff
@@ -23,16 +23,16 @@
 
 auto uri1 = uri_t("../experiments/data/c41078a1.tif");
 auto options1 = options_t{
-    "-of", "MEM",
+    "-of", "VRT",
```

Comment:
It has always been VRT; GDAL used the last appended -of to the param list. With MEM tests seem to be super slow btw.

Reply:
Interesting.  I'm okay with this change, but my recollection is that these should write to in-memory structures which should be relatively fast.